pkg/api/validation/validation.go

```diff
@@ -58,6 +58,8 @@ var DNS1123LabelErrorMsg string = fmt.Sprintf(`must be a DNS label (at most %d c
 var DNS952LabelErrorMsg string = fmt.Sprintf(`must be a DNS 952 label (at most %d characters, matching regex %s): e.g. "my-name"`, validation.DNS952LabelMaxLength, validation.DNS952LabelFmt)
 var pdPartitionErrorMsg string = IntervalErrorMsg(0, 255)
 var PortRangeErrorMsg string = IntervalErrorMsg(0, 65536)
+var GroupIdRangeErrorMsg string = IntervalErrorMsg(-1, 2147483648)
```

Comment:
> : Uids and gids must be in range 0-2147483647" refer the issue

@mdshuai that message comes from lib container, but I suppose the check is valid as long as libcontainer has that restriction. Perhaps add a comment to clarify that though. If you want you can file an issue against libcontainer and refer to it.